app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/ViewerFragment.java

```diff
@@ -376,4 +417,66 @@ private void onCreateGeographicMessage(PointGeometry pointGeometry) {
         // TODO: Update argument type and name
         void onFragmentInteraction(Uri uri);
     }
+
+    private static class ElapsedTimeUserLocationSymbolizer implements UsersLocationViewModel.UserLocationSymbolizer {
+
+        private static final Logger sLogger = LoggerFactory.create(
+                ElapsedTimeUserLocationSymbolizer.class);
+
+        //TODO: config , add these to the static configuration file after merging configuration feature branch
+        private static final long USER_LOCATION_STALE_THRESHOLD_MS = 60 * 1000;
+        private static final String ACTIVE_USER_LOCATION_PIN_COLOR_CSS_STRING = "#7FFF00";
+        private static final int USER_LOCATION_PIN_SIZE_PX = 48;
+        private static final String STALE_USER_LOCATION_PIN_CSS_COLOR = "#A9A9A9";
+
+        @Override
+        public Symbol symbolize(UserLocation userLocation) {
+            long timestampsDeltaMs = getTimestampsDeltaMs(userLocation);
+
+            sLogger.v(String.format("Symbolizing userLocation with id %s and timestamp delta %s",
```

Comment:
I cannot know at that point if its a new user/stale-user.
I did logged the type 
And anyways, its a verbose log. !completed